### PR TITLE
Make pages mobile responsive

### PR DESCRIPTION
**Pull request by dereklance**

### CODE REVIEW

File: src/app/components/shared-components.module.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CenteredLoadingSpinnerComponent } from './centered-loading-spinner/centered-loading-spinner.component';
+import { FooterComponent } from './footer/footer.component';
+
+
+
+@NgModule({
+  declarations: [CenteredLoadingSpinnerComponent],
```
Comment (aaronpxavier) [inline src/app/components/shared-components.module.ts]: There shouldn't be a nested module here. this should go inside components folder inside the shared module

**Review by aaronpxavier — CHANGES_REQUESTED** [review]
no need for extra nested module in root components should be inside shared module

File: src/app/manage-bookings/models/booking-types.ts [hunk 2]
```diff
@@ -1,14 +1,17 @@
 import { FlightModel } from '../../flight-booking/models/flight';
 
-export interface TravelerModel {
-  travelerId: number;
+export interface TravelerRequest {
   name: string;
   address: string;
   phone: string;
   email: string;
   dob: Date;
```
Comment (aaronpxavier) [inline src/app/manage-bookings/models/booking-types.ts]: can we move this into a models folder inside the shared module

**Review by aaronpxavier — CHANGES_REQUESTED** [review]

File: src/app/manage-bookings/models/booking-types.ts (hunk 2 above)
Comment (dereklance) [reply src/app/manage-bookings/models/booking-types.ts]: yes

File: src/app/components/shared-components.module.ts (hunk 1 above)
Comment (dereklance) [reply src/app/components/shared-components.module.ts]: I will refactor my modules in a new pull request

**Review by aaronpxavier — APPROVED** [review]

**Pull request merged**